activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/src/test/java/org/activiti/starter/tests/definition/ProcessDefinitionIT.java

```diff
@@ -86,9 +86,9 @@ public void shouldRetrieveListOfProcessDefinition() throws Exception {
         ParameterizedTypeReference<PagedResources<ProcessDefinition>> responseType = new ParameterizedTypeReference<PagedResources<ProcessDefinition>>() {
```

Comment:
@erdemedeiros we should ignore these changes because are pure formatting of stuff that doesn't exist anymore in master.